ksqldb-rest-app/src/test/java/io/confluent/ksql/rest/integration/PauseResumeIntegrationTest.java

```diff
@@ -236,7 +236,6 @@ private void createQuery(final String suffix) {
     TEST_HARNESS.getKafkaCluster().waitForTopicsToBePresent(SINK_TOPIC + suffix);
   }
 
-  @NotNull
```

Comment:
For some reason, spotbugs kept thinking this can return null. Looks to me like a false alarm, not why I'm getting it.